Dockerfile

```diff
@@ -96,13 +96,15 @@ RUN zypper addrepo http://download.opensuse.org/repositories/home:illuusio/openS
     iproute2 \
     kmod \
     libasan3 \
+    libICE6 libX11-6 libX11-data libX11-xcb1 libXau6 libXdamage1 libXext6 libXfixes3 libXft2 libXmu6 libXmuu1 libXrender1 libXss1 libXt6 libXxf86vm1 \
```

Comment:
Would it make sense to add a comment explaining that these are all X related things?